### DISCUSSION TITLE
How often do you have to use task manager to kill a stuck Maui build?

### DISCUSSION THREAD

**Discussion by martinwhitman** [dotnet/maui#26720] [General]
This is happening to me pretty frequently, like once an hour. I'm using VS on Windows, and a build will just hang, without stopping, without errors, without finishing, without printing any output to the console. I blamed my computer being old or RAM bound, but after upgrading to a brand new CPU and quadrupling my RAM I'm still having the same issues with the same frequency.

**Comment by the-gozo** [comment]
For me it was the antivirus software that caused long build times. IT department somehow was able to whitelist the build process. Now its ok.
